release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2590,3 +2590,29 @@ link:https://access.redhat.com/solutions/6815061[{product-title} 4.10.4 containe
 ==== Updating
 
 To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-5"]
+=== RHBA-2022:0982 - {product-title} 4.10.4 bug fix and security update
+
+Issued: 2022-03-21
+
+{product-title} release 4.10.5, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:0982[RHBA-2022:0982] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:0927[RHSA-2022:0927] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6828301[{product-title} 4.10.5 container image list]
+
+[id="ocp-4-10-5-known-issues"]
+==== Known issues
+
+* There is an issue adding a cluster through zero touch provisioning (ZTP) with the name `ztp*`. Add it causes a situation where `ArgoCD` deletes policies that ACM copies in to the cluster namespace. Consequently leading to a reconciliation loop, and the policies will not be compliant. As a workaround, do not name clusters with `ztp` at the beginning of the name. By adjusting the name of the cluster, collision will stop the reconciliation loop and the policies will be compliant. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2049154[*BZ#2049154*])
```

Comment:
Hi @opayne1 Replace the start of the second sentence with '**Adding** it causes a situation...'.

Reply:
updated